packages/devtools_app/lib/src/code_size/code_size_table.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../auto_dispose_mixin.dart';
+import '../charts/treemap.dart';
+import '../table.dart';
+import '../table_data.dart';
+import '../utils.dart';
+import 'code_size_controller.dart';
+
+class TreemapTreeTable extends StatefulWidget {
```

Comment:
I think we can make CodeSizeTable a stateless widget if we pass in the currentRoot as a parameter. See `CpuCallTreeTable` for an example of a stateless widget that builds a tree table

Reply:
Done